packages/hoppscotch-sh-admin/src/helpers/axiosConfig.ts

```diff
@@ -17,4 +17,9 @@ const restApi = axios.create({
   baseURL: import.meta.env.VITE_BACKEND_API_URL,
 });
 
-export { gqlApi, restApi };
+const listmonkApi = axios.create({
+  ...baseConfig,
+  baseURL: 'https://listmonk.hoppscotch.com/api/public',
```

Comment:
should we add the URL in the env file ?

Reply:
@nivedin No. I confirmed it with @AndrewBastin. It will be hardcoded.